src/components/FileUpload/FileUpload.tsx

```diff
@@ -44,7 +44,8 @@ export const FileUpload: React.FC<FileUploadProps> = ({
 	passwordIncorrectText = 'Invalid password. Please try again.',
 	passwordConfirmButtonText = 'Confirm',
 	passwordCancelButtonText = 'Cancel',
-	passwordPlaceholderText = 'Password'
+	passwordPlaceholderText = 'Password',
+	invalidFileFormatErrorMessage = 'Invalid file format. Please try a different one.'
```

Comment:
```suggestion
	unsupportedFileFormatErrorMessage = 'Unsupported file format. Supported formats:'
```

Also I would rename it because for example zip format is a VALID format, it's just NOT SUPPORTED by our file upload component 😊